activeadmin.gemspec

```diff
@@ -17,16 +17,16 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = %w[CHANGELOG.md CODE_OF_CONDUCT.md CONTRIBUTING.md README.md]
 
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.5'
```

Comment:
This would mean we drop support for users running ruby 2.4. We don't want to do that yet because ruby 2.4 is still a supported version.

Reply:
Yes, for Rails 6 and 7 leaving 2.4 was breaking to install.

Reply:
We have already dropped support separately so this should no longer be a problem.